src/renderer/store/reducers/index.ts

```diff
@@ -9,6 +10,7 @@ const rootReducer = combineReducers({
   recentProjects: recentProjectsReducer,
   currentPage: currentPageReducer,
   undoStack: undoStackReducer,
+  io: exportIoReducer,
```

Comment:
You need to change this, otherwise the whole thing throws an error.

Reply:
Whoops. Fixed now. Thanks for pointing that out.

Reply:
In relation to above comment regarding local testing of UI:

Not really. Another PR is coming with the export file menu option which will trigger the entire process.
If you want to see the export card you can edit the conditional rendering to be `true`.